test/ai/AiSetup.java

```diff
@@ -0,0 +1,221 @@
+package ai;
```

Comment:
Convenience class, pretty much all the ai test classes inherit this class and its methods